1-js/03-code-quality/04-ninja-code/article.md

```diff
@@ -223,6 +223,6 @@ Imagine, another developer wants only to check the email, and not output any mes
 
 All "pieces of advice" above are from the real code... Sometimes, written by experienced developers. Maybe even more experienced than you are ;)
 
-- Follow some of them -- and your code will become full of surprises.
-- Follow many of them -- and your code will become truly yours, no one would want to change it.
-- Follow all -- and your code will become a valuable lesson for young developers looking for enlightment.
+- Follow some of them, and your code will become full of surprises.
```

Comment:
Are you sure that a dash is wrong here? The phrases look better with it. Are the rules  really so strict that there must be no dash, or maybe a kind of "author punctuation" is possible?